data/json/recipes/basecamps/expansion/recipe_modular_workshop/recipe_modular_workshop_common.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "faction_base_workshop_0",
+    "description": "Survey land for a workshop.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "skill_used": "fabrication",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "faction_base_workshop_0",
+    "blueprint_name": "workshop survey",
+    "time": "3 h",
+    "blueprint_requires": [ { "id": "not_an_upgrade" } ],
+    "blueprint_provides": [ { "id": "fbmw_0" } ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "fbmw_1",
+    "description": "Select the blueprints for version 1 of the workshop. This uses a more sprawling style than version 2.",
```

Comment:
Yeah there needs to be 2 spaces between the sentences..

Reply:
I'll change it, of course, but the comment is still confusing, as JSON_STYLE.md doesn't seem to say anything about it, so I don't know why I would be expected to know about it. Also, out of curiosity, why? I assume it does have some technical reason, but what, and how does it manifest itself?

Reply:
> I assume it does have some technical reason, but what, and how does it manifest itself?

It's a standard for monospaced text. You may or may not have been told when you were younger to put two spaces between sentences, and that is why. Today we don't tend to use monospaced text, so it is not done in many contexts.

This standard is documented in `doc/MANUAL_OF_STYLE.md`
> Use double sentence spacing after periods.  This means that a period that ends a sentence should be followed by two spaces.  If the sentence is the last in the block of text, there should be no spaces following it.

Reply:
Thanks for the explanation. I've never heard of any such rule, despite definitely being old enough: the only thing I believe I've heard is to put exactly one space in between sentences (and not anything about exception for typewriters, which are inherently fixed size, though that education was both very brief and compressed further by circumstances). Anyway, those are the (local) rules, and so they should be followed.

Reply:
You can actually find switches for "one or two spaces after a period" in a lot of spelling/grammar checkers, BTW.